README.md

```diff
@@ -9,7 +9,7 @@ Rails application to track, audit and replicate archival artifacts associated wi
 ## Table of Contents
 
 * [Getting Started](#getting-started)
-    * [PostgreSQL](#postgreSQL)
+    * [PostgreSQL](#postgresSQL)
```

Comment:
i believe the correct spelling is `PostgreSQL`
https://www.postgresql.org/

Reply:
good 👀 - I was trying to fix that.  haha.  new commit and rebased.